images/win/scripts/Installers/Install-Go.ps1

```diff
@@ -29,36 +29,37 @@ function Install-GoVersion
 
     # Extract the zip archive.  It contains a single directory named "go".
     Write-Host "Extracting Go $latestVersion..."
-    Expand-Archive -Path $goArchPath -DestinationPath "C:\" -Force
+    $toolDirectory = Join-Path $env:AGENT_TOOLSDIRECTORY "go\$latestVersion"
+    7z.exe x $goArchPath -o"$toolDirectory" -y | Out-Null
```

Comment:
`-o"$toolDirectory"` - space missed

Reply:
I don't know why, but this flad uses without whitespace:
https://sevenzip.osdn.jp/chm/cmdline/switches/output_dir.htm

Reply:
Yep, missed space is expected here :)